### ISSUE TITLE
There are folder names with quotes present

### ISSUE THREAD

**Issue by arnavb** [OpenGenus/cosmos#4656]
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [x] Error
- [ ] Proposal to the Repository

**Details:**

The following folder names have quotes in them:

```
./code/quantum_algorithms/shor's_algorithm
./code/quantum_algorithms/shor's_algorithm/P1_shor_primefactorization.py
./code/quantum_algorithms/shor's_algorithm/README.md
./code/quantum_algorithms/grover's_algorithm
./code/quantum_algorithms/grover's_algorithm/README.md
./code/quantum_algorithms/grover's_algorithm/P1_grover_plot.py
```

This breaks various UNIX tools, such as `xargs` and as such needs to be fixed. If anyone would like to take on this issue please comment below.

**Comment by ritish099** [comment]
I want to take this issue ,assign it to me. I will correct the folder names

**Comment by Aman-Codes** [comment]
I am a GSSOC participant and I would like to work on this issue. Please assign me this issue.

**Comment by arnavb** [comment]
@Aman-Codes Since @ritish099 commented first, I will assign him the issue.

**Comment by ritish099** [comment]
check it out @arnavb

**Comment by arnavb** [comment]
Fixed in #4664
